content/projects/gsoc/2019/project-ideas/configuration-as-code-compatibility.adoc

```diff
@@ -0,0 +1,30 @@
+---
+layout: gsocprojectidea
+title: "Configuration as Code Compatibility"
+goal: "Improve Configuration as Code Compatibility in plugins"
```

Comment:
You have not taken the goal from the proposal

Reply:
I missed it. It's called "summary" in the proposal.